src/main/java/io/github/benas/jpopulator/impl/PopulatorImpl.java

```diff
@@ -27,8 +27,9 @@
 import io.github.benas.jpopulator.api.Exclude;
 import io.github.benas.jpopulator.api.Populator;
 import io.github.benas.jpopulator.api.Randomizer;
-import org.apache.commons.beanutils.PropertyUtils;
 import org.apache.commons.math3.random.RandomDataGenerator;
+import org.springframework.objenesis.Objenesis;
```

Comment:
Yes, I missed import update after changing pom dependency. Thanks for an update.